DemoQA/cypress/pages/bookstore/bookPage.js

```diff
@@ -0,0 +1,26 @@
+export const bookPage= {
+    TXT_SEARCH : "#searchBox",
+    TB_BOOK: '.ReactTable',
+    BTN_ADDBOOK: "[type='button']",
+    
+
+    selectBook(bookname){
+        cy.get(this.TB_BOOK).contains(bookname).click({force: true});
```

Comment:
Careful with force, it may pass some bugs that user cannot interact with element because it covered.